setup/install_upstream_rust_keylime/test.sh

```diff
@@ -53,16 +72,32 @@ _EOF'
         # Add drop-in update to enable logging output
         if [ -f /usr/lib/systemd/system/keylime_agent.service -o -f /etc/systemd/system/keylime_agent.service ]; then
             rlRun "mkdir -p /etc/systemd/system/keylime_agent.service.d"
+            DIREXIST="TRUE"
             rlRun "cat > /etc/systemd/system/keylime_agent.service.d/20-rust_log_trace.conf <<_EOF
 [Service]
 Environment=\"RUST_LOG=keylime_agent=trace\"
 _EOF"
             rlRun "systemctl daemon-reload"
         fi
-    rlPhaseEnd
+
+        id keylime && rlRun "chown -R keylime /var/tmp/limeLib && chmod -R g+w /var/tmp/limeLib"
+        grep -q LLVM_PROFILE_FILE /etc/bashrc || rlRun "echo 'export LLVM_PROFILE_FILE=/var/tmp/limeLib/coverage/rust_keylime_codecoverage-%p-%m.profraw' >> /etc/bashrc"
+            if [ $DIREXIST == "FALSE" ];then
+            rlRun "mkdir -p /etc/systemd/system/keylime_agent.service.d"
+            fi
+            rlRun "cat > /etc/systemd/system/keylime_agent.service.d/10-coverage.conf <<_EOF
+[Service]
```

Comment:
Not sure if Rust allows it but what we do for Python is that we define a unique context for every single test.. This way we can measure code coverage for individual tests. But I guess we can implement it later eventually, no need to rush.
Just for the record, relevant code pieces are in
https://github.com/RedHat-SP-Security/keylime-tests/blob/main/Library/test-helpers/lib.sh#L1711
https://github.com/RedHat-SP-Security/keylime-tests/blob/main/setup/enable_keylime_coverage/test.sh#L30

Reply:
Sure, I'll look more to this area after resolving simple code coverage of rust components.